apps/website/src/content/docs/list.mdx

```diff
@@ -34,9 +34,9 @@ There is also a `size` prop which can be set to `'large'` to ensure that all ite
 
 Please note that this prop only provides the hover styling and does nothing else. Usually you want to combine this prop either with `LinkAction` (which lets you use an anchor or a button) or with a more advanced component, such as [`Select`](select), which will do many additional things like set `role=option`, manage roving tabindex, and handle arrow-key navigation.
```

Comment:
There seem to be a few leftover references to the old `LinkAction` on this page. E.g.:

```suggestion
Please note that this prop only provides the hover styling and does nothing else. Usually you want to combine this prop either with `ListItem.Action` (which lets you use an anchor or a button) or with a more advanced component, such as [`Select`](select), which will do many additional things like set `role=option`, manage roving tabindex, and handle arrow-key navigation.
```

Reply:
fixed. [`9562e9c`](https://github.com/iTwin/iTwinUI/pull/1791/commits/9562e9ce6a7aed317e1ba3c3a985f2a77c39925c)